.github/workflows/ci.yaml

```diff
@@ -29,10 +29,18 @@ jobs:
         with:
           toolchain: stable
           override: true
+      - name: Install Rust Targets
```

Comment:
i wanted to have some working solution as this seems to be priority in my opinion now,
sure we can split build (then platform specific part, will be build in parallel), so it should be also faster, but also more complicated ...
but i prefer to merge this one first, and do improvements in next iteration/PR

Reply:
Why is it more complicated? Isn’t it simpler, just a duplication of the job instead of doing multiple architectures in a single one?
I don’t think there is anything that is not platform specific.

Reply:
ok, i will take o look on that

Reply:
I agree as well. Two different parallel tasks. Duplication here is more important than reuse

Reply:
I think, I can reason about this PR. After reading about buildx
https://github.com/golemcloud/golem-services/pull/59#discussion_r1468238014

Reply:
See if you can paralllelise it for two in separate jobs making it faster as well. But not super concerned as I was before

Reply:
So, I remembered wrong as I thought there is a separate job that builds the release executables, and another that takes them and builds the container(s). I guess that's in the `golem` repo not here. Confusing :) So now I understand that this was really the simplest way to add this.

I still think it would be better to not build two different archs in the same job, as all the rust compilation will not be able to share anything between them so it will be sequential in time, and take double disk space, etc. 

So if it's possible (I feel it's easy but maybe I'm overlooking something) I would do it the same way as in the other repo - build the two different release artifacts in parallel and then have a third github actions job that takes the result of both and builds the containers. It would also make the two CI pipelines more similar, which reduces confusion. 

I'm removing the "request for changes" so I don't block you but please consider doing it :)

Reply:
sure, thanks for comment, in next PR, i will try to do cargo build per each arch in parallel ... and then publish